atom/lib/getEnv.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const consistentEnv = require('consistent-env');
+
+const envPromise = consistentEnv.async()
+  .catch(error => {
+    atom.notifications.addError(error.message);
+    throw error;
+  });
+
+module.exports = () => envPromise;
```

Comment:
When you log in to a shell, there's a bunch of stuff that happens. Some of these things end up setting or modifying environment variables which are important for commands to be executed as you would expect. This consistentEnv package (which is used by atom linter) will get the correct ENV, consistently.

I originally wrote this so that the results would be cached, but I since realized that the package does this internally so this is unnecessary. I can simplify it some.